esphome/components/mcp4728/mcp4728_output.h

```diff
@@ -0,0 +1,85 @@
+#pragma once
+
+#include "esphome/core/component.h"
+#include "esphome/components/output/float_output.h"
+#include "esphome/components/i2c/i2c.h"
+
+namespace esphome {
+namespace mcp4728 {
+
+enum class CMD {
+  FAST_WRITE = 0x00,
+  MULTI_WRITE = 0x40,
+  SINGLE_WRITE = 0x58,
+  SEQ_WRITE = 0x50,
+  SELECT_VREF = 0x80,
+  SELECT_GAIN = 0xC0,
+  SELECT_POWER_DOWN = 0xA0
+};
+
+enum MCP4728Vref { MCP4728_VREF_VDD, MCP4728_VREF_INTERNAL_2_8V };
```

Comment:
It seems like these values are used as integers in `seq_write_`/`multi_write_`. In that case they should have explicit integer mappings to avoid mistakes

```suggestion
enum MCP4728Vref { 
	MCP4728_VREF_VDD = 0, 
	MCP4728_VREF_INTERNAL_2_8V = 1
 };
```

Same for other enums

Reply:
Changed. I have even changed other enums whose values were used as data or indexes.